lib/docs/core/scraper.rb

```diff
@@ -65,6 +65,10 @@ def root_url
       @root_url ||= root_path? ? URL.parse(File.join(base_url.to_s, root_path)) : base_url.normalize
     end
 
+    def home_url
+      @home_url ||= URL.parse self.class.home_url if self.class.home_url
```

Comment:
No need to parse the URI, we only ever use it as a string. We can just return `self.class.home_url`.